### PR TITLE
Move \n to Enter key

### PR DESCRIPTION
**Pull request by javigomez**
fixes issue #58

### CODE REVIEW

**Comment by yvesh** [comment]
+1

File: tests/acceptance/AdministratorWeblinksCest.php [hunk 1]
```diff
@@ -63,7 +63,8 @@ public function administratorCreateWeblinkTrash(AcceptanceTester $I)
 		$I->checkForPhpNoticesOrWarnings();
 
 		$I->amGoingTo('Search the just saved weblink');
-		$I->fillField(['id' => 'filter_search'], $this->title . "\n");
+		$I->fillField(['id' => 'filter_search'],$this->title);
```
Comment (puneet0191) [inline tests/acceptance/AdministratorWeblinksCest.php]:
```
$I->fillField(['id' => 'filter_search'], $this->title);

````
Update the Code Style please

File: tests/acceptance/AdministratorWeblinksCest.php [hunk 2]
```diff
@@ -63,7 +63,8 @@ public function administratorCreateWeblinkTrash(AcceptanceTester $I)
...
+		$I->pressKey(['id' => 'filter_search'],WebDriverKeys::ENTER);
```
Comment (puneet0191) [inline tests/acceptance/AdministratorWeblinksCest.php]: What stops us from doing a click on Search Button? something like this
```
$I->click(['xpath' => "//button[@type='submit' and @data-original-title='Search']"]);
```

File: tests/acceptance/AdministratorWeblinksCest.php (hunk 1 above)
Comment (javigomez) [inline tests/acceptance/AdministratorWeblinksCest.php]: see: https://docs.joomla.org/Testing_Joomla_Extensions_with_Codeception#Do_not_leave_spaces_between_parameters_in_Cepts

File: tests/acceptance/AdministratorWeblinksCest.php (hunk 2 above)
Comment (javigomez) [inline tests/acceptance/AdministratorWeblinksCest.php]: nothing, we can do that way too. Do you prefer the other way?

File: tests/acceptance/AdministratorWeblinksCest.php (hunk 1 above)
Comment (puneet0191) [inline tests/acceptance/AdministratorWeblinksCest.php]: :+1:  sounds good

File: tests/acceptance/AdministratorWeblinksCest.php (hunk 2 above)
Comment (puneet0191) [inline tests/acceptance/AdministratorWeblinksCest.php]: We can keep this way across all the tests, moreover is following the same approach as the rest of the test...

**Comment by puneet0191** [comment]
@javigomez I just merged your commit the way we discussed 
https://github.com/joomla-extensions/weblinks/commit/4b488d69440f7719596284199ca066d64990b0bf

Should I close this PR now?

**Comment by javigomez** [comment]
Thanks :)

**Pull request closed without merging**